File: common/src/main/java/com/hearlers/gateway/DefaultObjectMapper.java
package com.hearlers.gateway;

import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;
import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;

public class DefaultObjectMapper extends ObjectMapper {

    public DefaultObjectMapper() {
        super();

        this.setDefaultSetterInfo(JsonSetter.Value.forValueNulls(Nulls.AS_EMPTY));
        registerModule(new JavaTimeModule());

    }
}

File: common/src/main/kotlin/com/hearlers/gateway/DefaultObjectMapper.kt
package com.hearlers.gateway

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
import com.fasterxml.jackson.module.kotlin.KotlinModule

open class DefaultObjectMapper : ObjectMapper() {
        init {
            val kotlinModule = KotlinModule.Builder()
            .build()
            registerModule(JavaTimeModule())
            registerModule(kotlinModule)
        }
}